src/bitmessageqt/__init__.py

```diff
@@ -2708,15 +2708,8 @@ def network_switch(self):
 
     # Quit selected from menu or application indicator
     def quit(self):
-        '''quit_msg = "Are you sure you want to exit Bitmessage?"
-        reply = QtGui.QMessageBox.question(self, 'Message',
-                         quit_msg, QtGui.QMessageBox.Yes, QtGui.QMessageBox.No)
-
-        if reply is QtGui.QMessageBox.No:
-            return
-        '''
-
-        if self.quitAccepted:
+        """Quit the bitmessageqt application"""
```

Comment:
write comment rather than triple quotes

Reply:
Look precisely: I removed triple quotes and added docstring.